libraries/Microsoft.Bot.Builder.Dialogs/Prompts/ActivityPrompt.cs

```diff
@@ -189,7 +189,7 @@ public override async Task RepromptDialogAsync(ITurnContext turnContext, DialogI
         {
             var state = (IDictionary<string, object>)instance.State[PersistedState];
             var options = (PromptOptions)instance.State[PersistedOptions];
-            await OnPromptAsync(turnContext, state, options, true, cancellationToken).ConfigureAwait(false);
+            await OnPromptAsync(turnContext, state, options, false, cancellationToken).ConfigureAwait(false);
```

Comment:
>false [](start = 61, length = 5)

This is a behavior change which could change the behavior of bots.  Please make sure this is documented in the change list in a way for customers to know that this happened.